packages/core/src/V2/Node.ts

```diff
@@ -0,0 +1,41 @@
+import { Observable, Subscription, SubscriptionLike } from 'rxjs';
+
+export interface INodePorts {
+  getPort(portName: string): Observable<unknown>;
+}
+
+export interface ISourceNode {
```

Comment:
As we discussed, "Node" name is kind of already taken by the Diagram data structure.
We do have the Source/Operator/Watcher prefix here which is good.
As I understand it, now links will generate a operator node so we can handle counts, etc. Then, is it really a "_Node_" we are refering to?
We could consider a new name. Alternatively accept different "Node" concepts lives in three domains; ReactFlow, Diagram, Execution.

Reply:
Your suggestion is fantastic!
I believe renaming it to `element` might be a better choice.